Gemfile

```diff
@@ -40,7 +40,7 @@ gem 'simplecov', '~> 0.8'
 unless RUBY_VERSION == "1.9.2"
```

Comment:
@JonRowe thanks.
I have a question for your idea.
The error happened for only "jruby" environment. It did not happen for "jruby-*" environment.
So, does your idea mean that we can disable rubocop for all the jruby environment?

```
matrix:
  include:
    - rvm: jruby => failed
      env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false -Xcompat.version=2.0'
    - rvm: jruby-head => passed
      env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false'
    - rvm: jruby-18mode => passed
      env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false'
    - rvm: jruby => failed
      env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false'
    - rvm: jruby-9.1.2.0 => passed
      env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false'
```